app/javascript/figgy/figgy_boot.js

```diff
@@ -187,14 +188,27 @@ export default class Initializer {
     }
 
     $.fn.manage_fields = function(option) {
-        return this.each(function() {
-            var $this = $(this);
-            var data  = $this.data('manage_fields');
-            var options = $.extend({}, DEFAULTS, $this.data(), typeof option == 'object' && option);
+      return this.each(function() {
+        var $this = $(this);
+        var data  = $this.data('manage_fields');
+        var options = $.extend({}, DEFAULTS, $this.data(), typeof option == 'object' && option);
 
-            if (!data) $this.data('manage_fields', (data = new FieldManager(this, options)));
-        })
+        if (!data) $this.data('manage_fields', (data = new FieldManager(this, options)));
+      })
     }
     $('.multi_value.form-group').manage_fields();
   }
+
+  initialize_embargo_date_select() {
+    $('#embargo-date-picker').attr('placeholder', 'e.g. ' + new Date().toLocaleDateString());
+
+    $('#embargo_date_action').change(function() {
+      const selection = $(this).val();
+      if (selection === 'date') {
+        $('#embargo-date-picker').show();
+      } else {
+        $('#embargo-date-picker').hide();
```

Comment:
comment: This is a nice touch, hiding the date picker until "Input a date" is selected.